prowler/providers/iac/iac_provider.py

```diff
@@ -99,65 +161,20 @@ def run_scan(self, directory: str) -> List[CheckReportIAC]:
 
             reports = []
 
-            # Process failed checks
+            # Process all findings
             for finding in output:
```

Comment:
```suggestion
            for findings_type in output:
```

This will clarify that `checkov` output is divided by check_type:
```
(Pdb) output[1]["check_type"]
'github_actions'
(Pdb) output[0]["check_type"]
'terraform'
```

Reply:
The check type is the framework of Checkov, I will add the changes soon that handle it correctly